exercises/prime-factors/prime_factors_test.go

```diff
@@ -1,15 +1,15 @@
 package prime
 
-// Return prime factors in increasing order
-
 import (
 	"reflect"
+	"sort"
 	"testing"
 )
 
 func TestPrimeFactors(t *testing.T) {
 	for _, test := range tests {
 		actual := Factors(test.input)
+		sort.Sort(ascending(actual))
```

Comment:
IMHO, using [sort.Slice](https://golang.org/pkg/sort/#Slice) would make it simpler

Reply:
I tried in-lining the sorting funcs, but it gets verbose: 

```go
sort.Slice(actual, func(ii, jj int) bool {return actual[ii] < actual[jj]})
sort.Slice(test.expected, func(ii, jj int) bool {return test.expected[ii] < test.expected[jj]})
```

So I opted for a named func. Similar in LOC to the previous sort impl.

Reply:
thanks, a func sounds good.